hardhat/scripts/deploy.js

```diff
@@ -0,0 +1,25 @@
+async function main() {
+    const [deployer] = await ethers.getSigners();
+
+    console.log("Deploying contracts with the account:", deployer.address);
+
+    const balance = await deployer.getBalance();
+    console.log("Account balance:", balance.toString());
+
+    const Token = await ethers.getContractFactory("Token");
+    const token = await Token.deploy("BlahToken", "BLAH", 1000000);
```

Comment:
Lol, this by itself probably doesn't belong in OSHA; if this is part of the SW testing then it should live in that repo instead.